src/data/lexicon.json

```
@@ -17976,6 +17981,19 @@
           "minLength": 1,
           "pattern": ".*\\d{2,}.*"
         },
+        "area_under_area": {
```

@samandun Shouldn't it be `area_under_air?`
```suggestion
        "area_under_air": {
```